### PR TITLE
add quick_start for testing the quick start code; update readme

### PR DESCRIPTION
**Pull request by crookse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,126 +29,177 @@
...
-      Rhum.assertEquals(run(), false);
-    })
-  })
-})
+Rhum.asserts.assertEquals(true, true) // pass
+Rhum.asserts.assertEquals(true, false) // fail
 ```
 
-### `Rhum.assertEquals`
+### `Rhum.skip`
```
Comment (saragee3) [inline README.md:160]: oh this is interesting. what's a use case for skipping a written test?

Comment (crookse) [reply README.md:160]: In case you want to ignore it temporarily

Comment (crookse) [reply README.md:160]: We’re following this API: heres the issue: https://github.com/denoland/deno/issues/4092

Comment (crookse) [reply README.md:160]: There were times where tests failed in windows and we had to comment out the tests. This is the same approach but with a method.

**Pull request merged**